tests/assets/131-ns-deleted-when-bundle-deleted.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: fleet.cattle.io/v1alpha1
+kind: GitRepo
+metadata:
+  name: test-ns-deleted-when-bundle-deleted
+  annotations:
+    {}
+  labels:
+    {}
+  namespace: fleet-local
```

Comment:
Not required.

Reply:
Actually, line 9 (namespace: fleet-local) seems required. I will leave that one